t/01-html-checks.t

```diff
@@ -0,0 +1,106 @@
+#!perl
+
+use strict;
+use warnings;
+
+use File::Find;
+use Test::More;
+eval { use HTML::Lint; 1 } or plan skip_all => "HTML::Lint not available";
+
+my @on_disk = ();
+my $is_todo = '';
+
+sub collect {
+    return if $File::Find::name !~ m/\.html$/ || $File::Find::name =~ m/\/pod\//;
+
+    my $module = $File::Find::name;
+    push @on_disk, $module
+}
+find(\&collect, 'UI/');
+
+sub content_test {
+    my ($filename) = @_;
+    my $ui_header_used = 0;
+    $ui_header_used = 1 if $filename =~ m/UI\/lib\//;
+
+    my ($fh, @tab_lines, @trailing_space_lines);
+    open $fh, "<$filename";
+    while (<$fh>) {
+        push @tab_lines, ($.) if /\t/;
+        push @trailing_space_lines, ($.) if / $/;
+        $ui_header_used = 1 if /ui-header\.html/;
+    }
+    close $fh;
+
+TODO: {
+    local $TODO = 'LedgerSMB HTML files is still under development';
+    $is_todo = Test::More->builder->todo;
+    ok((! @tab_lines) && (! @trailing_space_lines),
+        "Source critique for '$filename'");
+    fail("Line# with tabs: " . (join ', ', @tab_lines))
```

Comment:
@ylavoie I'm thinking this fail() should probably be a [diag()](http://search.cpan.org/~exodist/Test-Simple-1.302031/lib/Test/More.pm#Diagnostics) ?

(Same with the fail 2 lines below.)